### PR TITLE
Setup instructions tabs 

### LINKED ISSUE #7: Make setup instructions nicer
We should have different tabs for windows/linux/mac and some code styling(maybe some plugin)
- we should also provide commands for downloading and installing on each os (cc @MakMuftic @mpetrun5 )

### PR DESCRIPTION
**Pull request by irubido**
Tab JS and style
Missing instructions for each tab option, Windows, Linux, Mac
Screenshot 2021-02-04 at 15 30 48

Closes #7

### CODE REVIEW

File: src/partials/setupInstructions.html [hunk 1]
```diff
@@ -4,23 +4,88 @@
...
+            </ul>
+        </div>
+    </nav>
+    <div class="container section">
+        <div id="windowsContainer" class="content-tab is-active">
+            <div class="box">
+                <h1>Windows</h1>
+                <h3 class="subtitle is-5">Load balancer</h3>
+                <ul>
+                    <li>Download latest release: <a target="_blank" href="https://github.com/NodeFactoryIo/vedran-daemon/releases" >github.com/NodeFactoryIo/vedran/releases</a></li>
```
Comment (mpetrunic) [inline src/partials/setupInstructions.html]:
```suggestion
                    <li>Download latest release from <a target="_blank" href="https://github.com/NodeFactoryIo/vedran-daemon/releases" >Github</a></li>
```

**Review by mpetrunic — COMMENTED** [review]
Can instructions be numbered.
Dow

File: src/partials/setupInstructions.html (hunk 1 above)
Comment (irubido) [reply src/partials/setupInstructions.html]: Screenshot 2021-02-05 at 13 02 50

Comment (mpetrunic) [reply src/partials/setupInstructions.html]: Seems like part of my comment was missing. I meant to say, if @MakMuftic or @mpetrun5  Could add you terminal commands for linux/mac to download and unzip with wget and zip tool or something (maybe even command to install unzip)

**Pull request merged**